app/views/admin/useremail.html.erb

```diff
@@ -15,9 +15,10 @@
          Include banned accounts after 2015 (careful: can result in server slowness for all users)
        </label>
    <% end %>
-
+   
+   <br>
    <% if @address %>
-   <h3>Searched for <%= @address %></h3> 
+   <h3>Searched for <%= @address %> {<%= @users.length %>}</h3>
```

Comment:
Hi, can we make these regular parentheses? `( )` Thanks!!

Reply:
Hi, i have done the required changes and Raised a PR.